app/views/workfloweditor.html

```diff
@@ -56,7 +56,7 @@ <h3 class="available-containers">
            ng-if="refreshingWorkflows">
         <span class="glyphicon glyphicon-refresh glyphicon-refresh-animate">
         </span>
-        Please wait, retrieving Github and/or Bitbucket workflows...
+        Please wait, retrieving Github, Gitlab and/or Bitbucket workflows...
```

Comment:
This is the general refresh all workflows message, so I think in this case is makes sense to have the and/or.